test/e2e/vclient.go

```diff
@@ -210,7 +210,7 @@ func (c *vcClient) getStoragePolicyID() (string, string, error) {
 
 func (c *vcClient) getClusterVirtualMachineImage() (string, error) {
 	// Better to use tkg.5, otherwise the image may not get ip
-	kubectlCmd := "kubectl get clustervirtualmachineimage -A | grep tkg.5 | tail -n 1 | awk '{print $1}'"
+	kubectlCmd := "kubectl get clustervirtualmachineimage -A | grep -E 'tkg.5|photon-5' | tail -n 1 | awk '{print $1}'"
```

Comment:
is this file has been changed in https://github.com/vmware-tanzu/nsx-operator/pull/1137/files?

Reply:
fixed